database/src/desktop/persistence/persistence_manager.cc

```diff
@@ -206,10 +206,8 @@ void PersistenceManager::DoPruneCheckAfterServerUpdate() {
           tracked_query_manager_->PruneOldQueries(*cache_policy_);
       PruneForestRef prune_forest_ref(&prune_forest);
       if (prune_forest_ref.PrunesAnything()) {
-        std::cout << __LINE__ << std::endl;
```

Comment:
what were these for?

Reply:
I dunno, I was probably debugging something at some point and left them in by mistake.

Reply:
Ok, cool.